test/ng/directive/ngOptionsSpec.js

```diff
@@ -1395,6 +1426,28 @@ describe('ngOptions', function() {
         });
       }).not.toThrow();
     });
+
+    it('should remove the "selected" attribute when the model changes', function() {
```

Comment:
Technically, the problem appears when you update the model (which then updates the view)